Installations-HowTos/How-To-Install-Spark-On-Windows.md

```diff
@@ -125,8 +125,16 @@ This should fire up the spark server on your computer. The output would look som
 ![windows_install_01](https://github.com/kiat/BigDataAnalytics/blob/master/Installations-HowTos/sceenshots/windows_install_01.PNG)
 Enter Ctrl+Z to close the spark session.
 
+Now Let us configure the spark-env to use python3.
+Create a new file named 'spark-env.cmd' in the 'C:\ProgramData\spark\spark-3.5.0-bin-hadoop3\conf' and add following lines so that spark-submit will use python3 (please modify to relevant path as per your system)
+```
+set PYSPARK_PYTHON=C:\Users\sande\anaconda3\python.exe
+set PYSPARK_DRIVER_PYTHON=C:\Users\sande\anaconda3\python.exe
+```
+Now close the file and save it.
+
 So, our installation is good, now let's create PySpark script in python and see how we can execute it on spark.
-  * First, let us create a simple python script ```TestPySpark.py``` that will just print some details about the spark instance. You can save this file at a path of your choice. For this example, I have the file stored at ```C:\Users\Subrat\Documents\CS777```<br/>
+  * First, let us create a simple python script ```TestPySpark.py``` that will just print some details about the spark instance. You can save this file at a path of your choice. For this example, I have the file stored at ```C:\Users\sande\SubrDocuments\CS777```<br/>
```

Comment:
we can use a generic folder on c instead of sticking with specific user

Reply:
updated with generic place holder